### PR TITLE
Add login button to perfherder

### PR DESCRIPTION
**Pull request by wlach**
[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1159)

### CODE REVIEW

File: ui/perf.html [hunk 1]
```diff
@@ -72,14 +72,18 @@
...
   <script src="js/controllers/perf/compare.js"></script>
   <script src="js/controllers/perf/graphs.js"></script>
   <script src="js/controllers/perf/alerts.js"></script>
+  <script src="js/directives/treeherder/persona.js"></script>
   <script src="js/directives/perf/compare.js"></script>
   <script src="js/perfapp.js"></script>
   <script src="js/filters.js"></script>
   <!-- endbuild -->
 
+  <script src="https://login.persona.org/include.js"></script>
```
Comment (jmaher) [inline ui/perf.html]: I see we do this include for regular treeherder, is this safe to have this requirement on an external service?

Comment (wlach) [inline ui/perf.html]: Presumably this is just so we can take advantage of fixes to persona without having to update treeherder.

If we can't trust persona.org, we're in a lot of trouble. :)

Comment (jmaher) [inline ui/perf.html]: I agree.  Just if persona.org is down for a couple hours the trees will
probably be closed.

On Wed, Nov 25, 2015 at 12:42 PM, William Lachance <notifications@github.com

Comment (edmorley) [inline ui/perf.html]: Even if the JS snippet was hosted on treeherder.m.o, the Persona IDP is still required for authentication, so people would still not be able to log in.

**Pull request merged**